web/js/tables.js

```diff
@@ -98,32 +98,32 @@ function displayCellValue(type, colName, value) {
 	//convert known states to human-readable strings, if not known show as is
 	if ((colName == "state") || (colName == "occupiedSense")) {
 		switch (type) {
-			case "turnouts":
+			case "turnout":
 				switch (value) {
 					case 0: return "unknown";
 					case 2: return "closed";
 					case 4: return "thrown";
 					case 8: return "inconsistent";
```

Comment:
@rhwood Is there a way to define these value descriptions (2 => "closed" for example) in the schema or elsewhere?